modules/ROOT/pages/create-an-api-configure-service-mesh.adoc

```diff
@@ -4,9 +4,30 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: service mesh, microservices, downloading, installing
 
-After you provision the adapter, you must create an API to bind to the service. 
+After you provision the adapter, you must create an API in Anypoint Platform to bind to the service. 
 
-To create an API, perform the following steps:
+== Methods for Creating an API
+
+Based on your requirements, you can create your API using either of the following methods:
+
+* <<create-api-AP, Create an API using Anypoint Platform>>
++
+If you want to create REST or SOAP APIs with OAS or WSDL definition, you use Anypoint Platform. You can manage your contracts and complex API configurations easily in Anypoint Platform.
+* <<create-api-ASM-CLI, Create an API using Anypoint Service Mesh CLI>>
++
+If you want to create a simple HTTP API with no contracts or definitions, use the auto-creation feature of Anypoint Service Mesh. When you choose this option, Anypoint Service Mesh creates the API for you in Anypoint Platform using its powerful CLI tool. You do not have to manually log in to Anypoint Platform or manage your API. 
```

Comment:
IMHO, It should talk about contracts here. You can use contracts even if you created your API with SM.

Reply:
Added the comment in the doc to address. Thanks!